### PR TITLE
[openstack] New Openstack Trove (DBaaS) plugin.

### PR DESCRIPTION
**Pull request by lyarwood**
https://wiki.openstack.org/wiki/Trove

Signed-off-by: Lee Yarwood <lyarwood@redhat.com>

### CODE REVIEW

**Comment by lyarwood** [comment]
An example report from a basic allinone env is here :
http://people.redhat.com/lyarwood/sos/sosreport-trove-aio.example.com-20150407133211.tar.xz

File: sos/plugins/openstack_trove.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2009 Red Hat, Inc., Lee Yarwood <lyarwood@redhat.com>
```
Comment (battlemidget) [inline sos/plugins/openstack_trove.py]: Can you update the copyright year here?

**Comment by battlemidget** [comment]
+1 barring one small nitpick

File: sos/plugins/openstack_trove.py (hunk 1 above)
Comment (lyarwood) [inline sos/plugins/openstack_trove.py]: ACK, thanks.

**Pull request closed without merging**